packages/gatsby-theme-moltin/gatsby-config.js

```diff
@@ -1,26 +1,26 @@
 const path = require(`path`)
 
 module.exports = themeOptions => {
-  const { client_id } = themeOptions
+  const { client_id, clientId } = themeOptions
 
   return {
     plugins: [
       {
         resolve: `@moltin/gatsby-source-moltin`,
-        options: { client_id }
+        options: { client_id: client_id || clientId },
```

Comment:
So, we should support both and phase out `client_id`.

Reply:
@notrab This should be reflected at [`gatsby-source-moltin`](https://github.com/moltin/gatsby-source-moltin) too.

Reply:
@ynnoj Yep! Done, will submit another PR later.